ecosystem/python/sdk/.gitignore

```diff
@@ -161,5 +161,7 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
+.idea/
 
+# testing/fidling script:
+playground.py
```

Comment:
I would likely revert this change for the purposes of the Pull request and leave it how it is.

Reply:
I can remove the `playground.py` and simply delete the file, but IntelliJ always inserts a `.idea` folder which contains IntelliJ project specific settings, unique to each user, so I think this should be included in the .gitignore.

Can read more [here](https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-#:~:text=Follow&text=2023%2006%3A46-,The%20.,history%20and%20currently%20selected%20configuration.).

Reply:
Yeah, fair.  I don't know who added the original comment around .idea.  Just let's remove playground.